client/js/widgets/card.js

```diff
@@ -139,12 +139,33 @@ class Card extends Widget {
               objectDiv.style.backgroundImage = mapAssetURLs(`url("${object.value}")`);
             }
             objectDiv.style.backgroundColor = object.color || 'white';
+          } else if (object.type == 'html') {
+            // Prevent input from going to frame.
+            objectDiv.style.pointerEvents = 'none';
+            objectDiv.setAttribute('sandbox', 'allow-same-origin');
```

Comment:
This `'allow-same-origin'` permission means that the sub-page can fetch resources as same-origin requests. We could probably avoid it by serving the fonts css and font files with CORS headers so that the frame can still be treated as cross origin. I'll look into adding this to the PR.

Reply:
Done, I restored the full sandboxing and added CORS headers to the fonts.css and fonts so the cross origin frame can still fetch them.